index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Rizwan Hasan Protfolio</title>
+    <link rel="stylesheet" href="flex.css">
+</head>
+<body>
+
+    <div class="app-wrap">
+
+    <div class="app-header">
+
+     
+
+        <div class="header"> <header> Rizwan Hasan Profolio</header></div>
+        <div class="menu"><img src="Image/Menu-bar.PNG" alt="qries"
+            width="45" height="50"></div>
+
+
+    </div>
+            <br>
+            <br>
+            <br>
+            <br>
+
+            <div class="app-homepage">
+
+                    <h1 class="name">Hey There.<br>
+                        I'm Rizwan</h1>
+                        
+                        <h2> I'am a Software Developer</h2>
+
+
+                        
+                        <br>
+                        <br>
+
+
+                        <h3>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                            don’t hestiate to contact me.</h3>
+                    
+
+                    <br>
+                    <br>
+                            <div class="media">
+                                <ul>
+                                    <li class="github"> <a href="https://github.com/Rizwan-learn">
+                                                 <img alt="qries" src="Image/GitHubLogo.PNG" 
+                                                 width="45" height="50"></a>
+                                    </li>
+                                    <li class="link"><a href="https://www.linkedin.com/in/rizwan-h-4b3096150/">
+                                        <img alt="qries" src="Image/Linkedinlogo.PNG"
+                                        width="45" height="50"></a>            
+                                    </li>
+                                    
+                                    <li class="twitter"><a href="https://twitter.com/RizwanH97">
+                                        <img alt="qries" src="Image/TwittterLogo.PNG"
+                                        width="45" height="50"></a>            
+                                    </li>                             
+
+                                </ul>
```

Comment:
- [ ]as an aspiring professional frontend developer let us Kindly fix the image paths as they look broken in the design, also you should do well to add the background image as it is in the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)
  
![image](https://user-images.githubusercontent.com/66097371/148423840-762f1099-61b8-407a-b510-7e45c4e8f085.png)